--- bcmatroska2-5.2.1/libmatroska2/CMakeLists.txt.orig	2022-11-29 11:25:07.000000000 +0100
+++ bcmatroska2-5.2.1/libmatroska2/CMakeLists.txt	2023-04-09 20:19:33.979579064 +0200
@@ -33,17 +33,17 @@ endif()
 
 string(REPLACE ";" " " LINK_FLAGS_STR "${LINK_FLAGS}")
 if(ENABLE_STATIC)
-	add_library(bcmatroska2 STATIC $<TARGET_OBJECTS:corec-objects> $<TARGET_OBJECTS:ebml2-objects> ${MATROSKA2_SOURCE_FILES})
-	set_target_properties(bcmatroska2 PROPERTIES OUTPUT_NAME bcmatroska2)
-	set_target_properties(bcmatroska2 PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
+	add_library(bcmatroska2-static STATIC $<TARGET_OBJECTS:corec-objects> $<TARGET_OBJECTS:ebml2-objects> ${MATROSKA2_SOURCE_FILES})
+	set_target_properties(bcmatroska2-static PROPERTIES OUTPUT_NAME bcmatroska2)
+	set_target_properties(bcmatroska2-static PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
 	if(NOT "${LINK_FLAGS_STR}" STREQUAL "")
-		set_target_properties(bcmatroska2 PROPERTIES LINK_FLAGS "${LINK_FLAGS_STR}")
+		set_target_properties(bcmatroska2-static PROPERTIES LINK_FLAGS "${LINK_FLAGS_STR}")
 	endif()
-	target_include_directories(bcmatroska2 PRIVATE ../libebml2 .)
+	target_include_directories(bcmatroska2-static PRIVATE ../libebml2 .)
 	if(LIBS)
-		target_link_libraries(bcmatroska2 ${LIBS})
+		target_link_libraries(bcmatroska2-static ${LIBS})
 	endif()
-	install(TARGETS bcmatroska2 EXPORT BcMatroska2Targets
+	install(TARGETS bcmatroska2-static EXPORT BcMatroska2Targets
 		ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 		PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
 	)
